db/seeds.rb

```diff
@@ -7,5 +7,9 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-10.times { User.create!(email: "affanghazali@gmail.com", password: "affantest", admin: "false", name: "Affan M. Ghazali",
-                        phone: "03039138186") }
+User.create!(email: "affanghazali@gmail.com", password: "affantest", admin: "true", name: "Affan M. Ghazali", phone: "03039138186")
```

Comment:
You can use Figaro gem or each_with_index method for multiple users.
10.times.each_with_index {|index| puts "abc#{index}"}